packages/extension/src/components/pages/Nfts/Nfts.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useContext, useEffect, useState } from 'react';
+
+import { AccountNFTokenResponse } from '@gemwallet/constants';
+
+import { LedgerContext } from '../../../contexts';
+import { NftListing } from '../../organisms/NftListing';
+import { PageWithHeader } from '../../templates';
+
+const initalState = {
+  account_nfts: [],
+  marker: null
+};
+
+export const Nfts: FC = () => {
+  const { getNFTs } = useContext(LedgerContext);
+
+  const [nfts, setNfts] = useState<AccountNFTokenResponse>(initalState);
+
+  const fetchNfts = async () => {
```

Comment:
This can be defined within your useEffect.
If you let it here, the function is gonna be re-defined at each render of the component.
See an example here: https://github.com/GemWallet/gemwallet-extension/blob/master/packages/extension/src/components/pages/SendPayment/PreparePayment/PreparePayment.tsx#L55

Reply:
Here we need to pass this function to the next component as well so defined it outside the useEffect